src/cli/cli.ts

```diff
@@ -136,7 +136,7 @@ export class Cli {
         require('dotenv').config();
 
         for (let envVar in this.envVariables) {
-            let value = process.env[envVar] || process.env[`SOKETI_${envVar}`] || null;
```

Comment:
I think I'll prepare for 1.x by not taking into account any non-`SOKETI_` prefixed variable. One of the issues can be this: https://github.com/soketi/soketi/issues/453, where having an .env that may contain Soketi environments by accident (in this case `CACHE_DRIVER`) would lead to issues.

I think we should take into account just `SOKETI_*` variables. `master` branch will become 1.x